autosubmitconfigparser/config/configcommon.py

```diff
@@ -1033,6 +1033,8 @@ def _get_keys(
             keys = parameters.get(str(dynamic_var[0][start_long:-1]), None)
             if not keys:
                 keys = parameters.get(str(dynamic_var[0]), None)
+                if type(keys) in [int, float]:
+                    keys = None
```

Comment:
Maybe we can skip entering in this if altogether then?

```suggestion
            if not keys and type(keys) not int [int, float]:
                keys = parameters.get(str(dynamic_var[0]), None)
```